UnityProject/Assets/Scripts/Systems/Score/RoundEndScoreBuilder.cs

```diff
@@ -118,18 +154,40 @@ public void CalculateScoresAndShow()
 
 			foreach (var entry in ScoreMachine.Instance.Scores)
```

Comment:
I can't leave a suggested change (thx GitHub), but you should be able to replace the foreach with this:
```csharp
			foreach (var pair in ScoreMachine.Instance.Scores)
			{
				var entry = pair.Value;
				var score = entry switch
				{
					ScoreEntryInt a => a.Score,
					ScoreEntryBool m => m.Score ? boolScore : -boolScore,
					_ => 0,
				};
				entry.ScoreValue = score;
				switch (entry.Category)
				{
					case ScoreCategory.StationScore:
						stationScoreEntries.Add(entry);
						finalStationScore += score;
						break;
					//TODO: Add Antag Score UI.
					case ScoreCategory.AntagScore:
						antagScoreEntries.Add(entry);
						finalAntagScore += score;
						break;
				}
			}
```